requirements.txt

```diff
@@ -1,8 +1,9 @@
+black==20.8b1
 flake8==3.8.4
+ipapy==0.0.9.0
```

Comment:
For packages needed only by the code under `data/`, there's [`data/requirements.txt`](https://github.com/kylebgorman/wikipron/blob/baa00c1f8d653893c8da89f07445d2a46523315c/data/requirements.txt), not the repo top-level `requirements.txt` for the `wikipron` package and CircleCI builds.

Reply:
Done.